### DISCUSSION TITLE
Map get lost working on backup image

### DISCUSSION THREAD

**Discussion by devtechk** [mrbungle64/ecovacs-deebot.js#429] [Q&A]
### The question

Hello, I've got a Deebot N8 sometimes happen that the map disappear for one reason or another. I'm looking for a way to save and reload a previuos backed up Map. Ther's someone of you that have worked on this feature?
Another problem with map and deebot is when it goes automatic to create a new one Map then it finish and go to recharge, you can see the map created in the "automatic" tab in the app but nothing in the "Area" tab. So I'm looking for a way to set the map with an ID as default map created but with no success.
Can anyone help me please?
Thank you!
### Which model do you own?

Deebot N8 series

**Comment by mrbungle64** [comment]
@devtechk 

This library doesn't have a backup function for the map.
But current models usually have a backup function in the Ecovacs Mobile App.
Have you already checked if this is the case for your device?
